include/wil/filesystem.h

```diff
@@ -341,7 +341,7 @@ namespace wil
             {
 #if (NTDDI_VERSION >= NTDDI_WIN10_RS1)
                 // DeleteFile and RemoveDirectory use POSIX delete, falling back to non-POSIX on most errors. Do the same here.
-                FILE_DISPOSITION_INFO_EX fileInfoEx{};
+                FILE_DISPOSITION_INFO_EX fileInfoEx;

```

Comment:
I'm assuming this change is because all fields in the struct are written so the zero-init is unnecessary.  I'm not sure how likely that is to remain true or if this struct will ever change.  If the struct does increase in size then the zero-init would be missing.

Reply:
It does not increase in size though.

Reply:
Very unlikely to change though.

Reply:
There is also line 353 with the same change.

Reply:
What I said still applies though. There is nothing showing this struct is ever likely to change because ABI.

Reply:
Though perhaps it doesn't affect the ABI, but I have never seen this change specifically.